commit/merkleroot/outcome.go

```diff
@@ -177,7 +175,9 @@ func buildReport(
 					cciptypes.SeqNum(laneUpdate.ClosedInterval.MaxMsgNr),
 				),
 				MerkleRoot: cciptypes.Bytes32(laneUpdate.Root),
-				// TODO: add OnRamp as a string?
+				// NOTE: this hex encoding is just for the sake of giving the onramp address
+				// a value in this key struct.
+				OnRampAddress: hex.EncodeToString(laneUpdate.LaneSource.OnrampAddress),
```

Comment:
This is a plain `[]byte` (from the protobuf definition I believe)

Reply:
Shouldn't this use `typeconv.AddressBytesToString)`?

Reply:
The note is also confusing. All assignments are just for the sake of passing a value to the struct, why do you call this one out?

Reply:
Simply because its a transformation of the original data, which can't be used in the mapset since its not comparable.

If this is confusing I can hash it instead, I think we do that for exec.